protostar/cairo/cairo_bindings.py

```diff
@@ -33,7 +33,7 @@ def compile_starknet_contract_to_casm_from_path(
     input_path: Path,
     output_path: Optional[Path] = None,
     cairo_path: Optional[list[Path]] = None,
-) -> Optional[str]:
+) -> str:
```

Comment:
These binding functions still return optionals, this looks like an error to me - what if `cairo_python_bindings.compile_starknet_contract_to_casm_from_path` returns `None` - OK I checked and these functions never do but we should merge a PR with correct types in the compiler.

I think this should be addressed separately because there will be inconsistencies in the bindings. Could you please revert this change here and we can create an issue for this wdyt?

Reply:
https://github.com/software-mansion-labs/cairo/pull/71

Reply:
This PR should resolve this